tools/slicec-cs/src/slicec_ext/type_ref_ext.rs

```diff
@@ -107,3 +108,15 @@ fn dictionary_type_to_string(dictionary_ref: &TypeRef<Dictionary>, namespace: &s
             )
     }
 }
+
+// Helper method to convert a result type into a string
```

Comment:
Might as well make this a doc comment so it'll show up on hover.
```suggestion
/// Helper method to convert a result type into a string
```